src/DataLakeStore/DataLakeStore.Test/ScenarioTests/AdlsTests.ps1

```diff
@@ -699,16 +699,15 @@ function Test-DataLakeStoreFileSystem
 		Assert-AreEqual $result.FileCount 1
 
 		# Export DiskUsage
-		$targetFile = "/DuOutputFile"
-		Export-AdlStoreChildItemProperties -Account $accountName -Path $summaryFolder -OutputPath $targetFile -GetDiskUsage -IncludeFile -SaveToAdl
-		$result = Get-AzDataLakeStoreItem -Account $accountName -path $targetFile
+		$targetFile = "./ScenarioTests/DuOutput"
```

Comment:
Generally, you want to be careful to use the correct base directory (based on the executing assembly) to avoid issues between tests when using paths.  ALso, use Join-Path to join path segments - this guarantees that your path uses the correct platform-specific separators

Reply:
Assuming this specifies a local path and not an ADLS path

Reply:
fixed.